drafts/use-cases/3.obtaining-events.md

```diff
@@ -0,0 +1,110 @@
+## Entry point
+
+### Story
+As an application user
+I want to display all my events
+So I can find out what awaits me in the nearest future.
```

Comment:
Oh, I'd insist on a specific story. I believe that being generic in our current approach led us to nowhere.